.eslintrc

```
@@ -0,0 +1,46 @@
+{
+    "extends": [
+        "@eslint-kit/base",
+        "@eslint-kit/typescript",
+        "@eslint-kit/node",
+        "@eslint-kit/react",
+        "@eslint-kit/react-new-jsx-transform",
+        "@eslint-kit/prettier",
```

```suggestion
        "@eslint-kit/react",
```
Этих пакетов нет в зависимостях